src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -276,15 +268,16 @@ public static void start(Method testMethod) {
 
   private void stopLogging() {
     driver = DriverProvider.getActiveDriver();
-    if ( driver.getProxy() != null && Configuration.getForceHttps()) {
+    if (driver.getProxy() != null && Configuration.getForceHttps()) {
       Har har = driver.getProxy().getHar();
       for (HarEntry entry : har.getLog().getEntries()) {
         URL url = null;
         try {
           url = new URL(entry.getRequest().getUrl());
           if (url.getHost().contains("wikia")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS134)